test/gui/shared/scripts/helpers/api/sharing_helper.py

```diff
@@ -45,3 +49,30 @@ def has_public_link_share(user, resource_name):
             if share["file_target"].strip('\/') == resource_name:
                 return True
     return False
+
+
+def download_last_public_link_resource(user, resource, public_link_password=None):
+    last_stime = 0
+    share = False
+
+    public_link_shares = get_public_link_shares(user)
+
+    for share_link in public_link_shares:
+        if last_stime < share_link["stime"]:
+            share = share_link
+            last_stime = share_link["stime"]
+    if not share:
+        print("Could not be found")
```

Comment:
```suggestion
        print("Could not be found")
```
Can we make a message more clear? What could not be found?